### PR TITLE
fix usage of eth-tester import in production

### PR DESCRIPTION
**Pull request by weilbith**
See commit message for details.

### CODE REVIEW

**Comment by weilbith** [comment]
Ah, damn. We need to enable the CI pipline for the `arbitrum` branch now.

**Comment by weilbith** [comment]
I just updated the RSB for Arbitrum Rinkeby with this branch and it solves the issue as expected. Though I haven't yet tested a balance proof update yet. Is on the list. But it is not guaranteed that this code will run if the clock is behind the chain or in sync.

**Comment by karlb** [comment]
> Ah, damn. We need to enable the CI pipline for the `arbitrum` branch now.

I included the relevant commit in https://github.com/raiden-network/raiden-services/pull/1158. I can split it out into a separate PR, if you like.

File: src/monitoring_service/handlers.py [hunk 1]
```diff
@@ -533,8 +532,8 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
             )
         )
     except Exception as exc:  # pylint: disable=broad-except
-        error_message = exc.args[0]
-        if isinstance(exc, TransactionFailed) and "not allowed to monitor" in error_message:
+        error_message = exc.args[0] if len(exc.args) > 0 else ""
```
Comment (karlb) [inline src/monitoring_service/handlers.py:535]: I always wonder why there is no `list.get` method like there is for dicts. Would be nice, here.

**Review by karlb — APPROVED** [review]
Not the cleanest solution in the world, but simple and should be good enough.

**Comment by weilbith** [comment]
> Not the cleanest solution in the world, but simple and should be good enough.

I'll rebase on it when merged. 👍🏾

File: src/monitoring_service/handlers.py (hunk 1 above)
Comment (weilbith) [reply src/monitoring_service/handlers.py:535]: I am not aware of such a function. But yes, I would also love to have it. Do you know of any other solution. Haven't written much Python code in a long time.

Comment (karlb) [reply src/monitoring_service/handlers.py:535]: It's either your way or the "easier to ask forgiveness than permission" approach of
```python
try:
    error_message = exc.args[0]
except IndexError:
    error_message = ""
```
So no really elegant solution that I know of.

Comment (weilbith) [reply src/monitoring_service/handlers.py:535]: Okay, than I would rather go with the current state if that is okay. It is only a single line and good enough to read. Nothing advanced.

**Pull request merged**